docs/cookbook/cloudways.rst

```diff
@@ -0,0 +1,93 @@
+How to deploy Sylius to Cloudways PHP Hosting?
+==============================================
+
+Cloudways is a managed hosting platform for custom PHP apps and PHP frameworks such as Symfony, Laravel, Codeigniter, Yii
+CakePHP etc. You can launch the servers on any of the five providers including DigitalOcean, Vultr, AWS, GCE and KYUP containers.
+
+The deployment process of Sylius on Cloudways is pretty much straight forward and easy.
+
+Now to install Sylius you need to go through series of few steps:
+
+1. Launch Server with Custom PHP App
+------------------------------------
+
+You should `signup at Cloudways <https://platform.cloudways.com/signup>`_ to buy the PHP servers from the above mentioned providers. Simply go to the pricing page and choose your required plan.
+You then need to go through the verification process. Once it done login to platform and launch your first Custom PHP application. You can follow the Gif too.
+
+.. image:: ../_images/cloudways-php-server.gif
+    :align: center
+
+Now let’s start the process of installing Sylius on Cloudways.
+
+2. Install Sylius v1.0.0 via SSH
+--------------------------------
+
+Open the SSH terminal from the **Server Management tab**. You can also use PuTTY for this purpose. Find the SSH credentials under 
+the **Master Credentials** heading and login to the SSH terminal:
+
+.. image:: ../_images/cloudways-ssh.png
+    :align: center
+
+After the login, move to the application folder using the ``cd`` command and run the following command to start installing Sylius:
+
+.. code-block:: bash
+
+    $ composer create-project -s beta sylius/sylius-standard myshop
+
+The command will start installing the long list of dependencies for Sylius. Once the installation finishes, Sylius will ask for the database credentials.
+You can find the database username and password in the Application Access Details.
+
+.. image:: ../_images/cloudways-database.png
+    :align: center
+
+Enter the database details in the SSH terminal:
+
+.. image:: ../_images/cloudways-sylius-database.png
+    :align: center
+
+Keep the rest of the values to default so that the config file will have the defaults Sylius settings.
+If the need arises, you can obviously change these settings later.
+
+
+3. Install Node Dependencies
+----------------------------
+
+Sylius requires several Node packages, which also needs to be installed and updated before setting up the shop. In addition, I also need to start and setup gulp.
+
+Now move to the myshop folder by using ``cd myshop`` and run the following command ``yarn install``. Once the command finishes, run the next command, `yarn run gulp`.
+
+4. Populate the Database and Sylius
+-----------------------------------
+
+Comment out the three lines in ``config_dev.yml`` file.
```

Comment:
I'm not sure about this one, IMHO install command should be run with `-e prod` instead.